scripts/remove_after_use/node_preprint_share.py

```diff
@@ -0,0 +1,21 @@
+from website.app import setup_django
+setup_django()
+from osf.models import Preprint
+from website.preprints.tasks import update_preprint_share
+import progressbar
+
+ # To run: docker-compose run --rm web python -m scripts.remove_after_use.node_preprint_share
+def main():
+    """
+    Updates all preprints in SHARE post-divorce
+    """
+    preprints = Preprint.objects.all()
```

Comment:
I don't know why it is out of sync either @aaxelb! I'm also not sure of the scope of the issues.  I have some good OSF preprints that are showing up, others exactly like them that are not there.  Not sure how to distinguish the two. 

It looked like maybe an access token was missing for at least one provider, so maybe they're not set up for all the providers.  However, I'm specifically looking at OSF preprints.

Reply:
It could be a SHARE-side issue. Especially since all the stagings feed the same staging SHARE, there can be confusing contradictions that might cause SHARE to choke, like a staging1 user having multiple emails that belong to separate users on staging3. It tries to reconcile all that into a single truth, and sometimes fails.

If there are still missing preprints after running this script and someone gives me guids, I can look into what actually happened.

Reply:
OK!

Before running the script: 

guids showing up under staging3 osf preprints SHARE search, that shouldn't be there.  I believe these preprints were wiped by Steve
whmyu - Test Preprint Project
v26qy - Tried and True Title but edited

Guids that are missing that I would expect to see in SHARE staging3 preprint search
km6wy - Wolfberry
23wmq - Acai Berry child
4nud9 - Acai Berry
p7aru - Alpha Preprint
2tm7b - ORPHAN - Counting by Ones